eth_keys/backends/coincurve.py

```diff
@@ -67,3 +71,17 @@ def private_key_to_public_key(self, private_key: PrivateKey) -> PublicKey:
             compressed=False,
         )[1:]
         return PublicKey(public_key_bytes, backend=self)
+
+    def decompress_public_key_bytes(self,
+                                    compressed_public_key_bytes: bytes) -> bytes:
+        public_key = self.keys.PublicKey(compressed_public_key_bytes)
+        return public_key.format(compressed=False)[1:]
+
+    def compress_public_key_bytes(self,
+                                  uncompressed_public_key_bytes: bytes) -> bytes:
+        point = (
+            big_endian_to_int(uncompressed_public_key_bytes[:32]),
+            big_endian_to_int(uncompressed_public_key_bytes[32:]),
```

Comment:
This allows for an arbitrarily long second value were the provided `uncompressed_public_key_bytes` to be longer than `64` bytes.  Am I missing something?  Seems like this would cause weird/hard-to-debug downstream errors in some cases.

Reply:
This is not part of the public API, all inputs should pass through `datatypes.PublicKey`. I still added validation now, just to be safe.